src/tests/test_app.rs

```diff
@@ -91,67 +91,71 @@ where
     val.amount
 }
 
+/// Utility function for generating user addresses.
+fn addr_make(addr: &str) -> Addr {
+    MockApi::default().addr_make(addr)
+}
+
```

Comment:
consider using `into_addr` instead

Reply:
Yes, sure, that's what we designed this feature for ;-).